### DISCUSSION TITLE
Tool change in first sequence

### DISCUSSION THREAD

**Discussion by JouirnyMan011** [TimPaterson/Fusion360-Batch-Post#48] [Ideas]
Hi Tim,
First of all, thank you very much for this very nice piece of software. It helps tremendously. 

I am running the UCCNC with 2017 Screenset and everything is working as it is intend to do.  At the beginning, I load the G code and it is noted that there is a M6 within the code, which means I have to do the Z Auto Zero Initialize (if I don't do it, program will not run). Basically, Z Auto Zero Initialize is doing auto zero from the stock and then from the fixed location - to have a reference for other tools.  Then, when I run the code, at the beginning of the first sequence there is a line with T3 M6 which ask me to change the tool. Well, I don't need to do that, as this is the first tool used and it is already in the holder. My question is: is there an option to put this line of the code (T# M6) from the first sequence only as a comment automatically - when G code is generated or I have to make an edit and do it manually? Am I doing something wrong?
Thanks,
Milos

**Comment by TimPaterson** [comment]
You will need  to edit the G-code manually.

**Reply by JouirnyMan011** [reply]
Thanks Tim, This is not a problem.

**Comment by gcormier** [comment]
My fork has this option - https://github.com/gcormier/Fusion360-Batch-Post
